### ISSUE TITLE
The optimizeImages configuration param is ignored in production

### ISSUE THREAD

**Issue by Angelmmiguel** [cyrilwanner/next-optimized-images#110]
Hello,

I was checking the `optimizeImages` param to avoid the optimization process in certain builds of the project. After setting this property to `false`, the images were still optimized. 

In the `lib/index.js` file, it checks if the optimization process should be triggered in the current step:

```
// https://github.com/cyrilwanner/next-optimized-images/blob/master/lib/index.js#L30
// check if it should optimize images in the current step
const optimizeInCurrentStep = nextComposePlugins && typeof nextComposePlugins.phase === 'string'
  ? (
    nextComposePlugins.phase === 'phase-production-build'
    || nextComposePlugins.phase === 'phase-export'
    || (nextComposePlugins.phase === 'phase-development-server' && optimizeImagesInDev)
  )
  : (!dev || optimizeImagesInDev);
```

In this conditional, the lib only checks the `optimizeImagesInDev` property and `optimizeImages` is ignored. In fact, if `optimizeImages` is `false`, `optimizeInCurrentStep` must be `false` always.

I'll create a PR that takes into account the `optimizeImages` in this process.

**Comment by cyrilwanner** [comment]
Has been fixed in `v2.5.4`

**Comment by Angelmmiguel** [comment]
I created a new PR to fix this issue. The first one was working only when the project uses `next-compose-plugins`.

Thanks for your help and happy new year! 😄
